setup.py

```diff
@@ -321,7 +321,9 @@ def initialize_options(self) -> None:
           # Not sure how/if interface will change in db-facts, so
           # let's be conservative about what we're specifying for now.
           'db-facts>=4,<5',
-          'chardet',
+          # Version 4.0.0 is not yet compatible with requests, and
```

Comment:
This file, one of the 3 largest in the project, increased in size to 375 lines.  The total size of those files is now 3417 lines (target: 1143).  Is this file complex enough to refactor?